llvm/lib/Analysis/LazyValueInfo.cpp

```diff
@@ -1159,6 +1162,49 @@ getRangeViaSLT(CmpInst::Predicate Pred, APInt RHS,
   return std::nullopt;
 }
 
+static bool matchBitIntrinsic(Value *LHS, Value *Val, Intrinsic::ID &IID) {
```

Comment:
Unused?

Reply:
Ah, sorry. I forgot to remove it. Fixed.